cosmo_tester/config_schemas/cfy_manager_upgrade.yaml

```diff
@@ -0,0 +1,7 @@
+namespace: upgrade
+manager_image:
```

Comment:
This should not be here, we already have a list of images elsewhere- please use them.

Reply:
Note that you can see the usage of this between the snapshot tests' __init__.py and conftest.py

Reply:
We do have a list of images [here](https://github.com/cloudify-cosmo/cloudify-system-tests/blob/master/cosmo_tester/config_schemas/manager_image_names_rhel.yaml), but it doesn't include the 5.1.0-installer. 

Using the [5.1.0 image](https://github.com/cloudify-cosmo/cloudify-system-tests/blob/master/cosmo_tester/config_schemas/manager_image_names_rhel.yaml#L8) will fail because of the [issue described in RD-868](https://cloudifysource.atlassian.net/browse/RD-868). 

Should I instead add a 5.1.0-installer to `manager_image_names_rhel.yaml`?

Reply:
How about updating the config.yaml on the 5.1.0 image so that the broken entries are correct?

Reply:
I talked with @mabuaisha and this is not possible.

Reply:
In any case, I'll add the 5.1.0-installer to `manager_image_names_rhel.yaml` and `manager_image_names_centos.yaml` for future use.